usr.sbin/devinfo/devinfo.c

```diff
@@ -168,8 +270,13 @@ print_device(struct devinfo_dev *dev, void *arg)
 		}
 	}
 
-	return(devinfo_foreach_device_child(dev, print_device,
+	int ret = (devinfo_foreach_device_child(dev, print_device,
```

Comment:
style(9)  (man style) says that ret should be declared at the start of the block.